app/uk/gov/hmrc/thirdpartyorchestrator/domain/models/applications/Access.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.thirdpartyorchestrator.domain.models.applications
+
+import play.api.libs.json._
+
+import uk.gov.hmrc.apiplatform.modules.applications.domain.models.{PrivacyPolicyLocation, TermsAndConditionsLocation}
+import uk.gov.hmrc.thirdpartyorchestrator.domain.models.applications.AccessType._
+
+sealed trait Access {
+  val accessType: AccessType.Value
+}
+
+case class ImportantSubmissionData(
+    organisationUrl: Option[String] = None,
+    responsibleIndividual: ResponsibleIndividual,
+    serverLocations: Set[ServerLocation],
+    termsAndConditionsLocation: TermsAndConditionsLocation,
+    privacyPolicyLocation: PrivacyPolicyLocation,
+    termsOfUseAcceptances: List[TermsOfUseAcceptance]
+  )
+
+object ImportantSubmissionData {
+  implicit val format = Json.format[ImportantSubmissionData]
+}
+
+case class Standard(
```

Comment:
Are the classes in this package copied from TPA? It feels like they should be in `api-platform-application-domain`

Reply:
Pending app lib